webapp/src/main/resources/public/js/components/workbench/ViewModeDropdown.js

```diff
@@ -0,0 +1,40 @@
+import _ from "lodash";
+import React from "react";
+import createReactClass from 'create-react-class';
+import {FormattedMessage, injectIntl} from 'react-intl';
+import {DropdownButton, MenuItem, InputGroup, FormControl, Button, Glyphicon} from "react-bootstrap";
+import DateTime from "react-datetime";
+import FluxyMixin from "alt-mixins/FluxyMixin";
+import moment from 'moment';
+import keycode from 'keycode';
+
+import SearchParamsStore from "../../stores/workbench/SearchParamsStore";
+import SearchConstants from "../../utils/SearchConstants";
+import WorkbenchActions from "../../actions/workbench/WorkbenchActions";
+import ViewModeStore from "../../stores/workbench/ViewModeStore";
+
+let ViewModeDropdown = createReactClass({
+    renderModeMenuItem(mode) {
+        return (
+            <MenuItem eventKey={mode} active={this.props.viewMode === mode} onSelect={this.props.onModeSelected}>
+                <FormattedMessage id={"search.viewMode." + mode} />
+            </MenuItem>
+        );
+    },
+
+    render() {  
+        return (
+            <DropdownButton  
+                id="WorkbenchStatusDropdown" 
+                title={this.props.intl.formatMessage({id: "search.viewMode.title"})}
```

Comment:
Maybe update the dropdown title with the selected mode?

Reply:
Done.